tests/bot/cogs/moderation/test_utils.py

```diff
@@ -286,6 +265,8 @@ def setUp(self):
 
                 if not error:
                     self.bot.api_client.post.assert_awaited_once_with("bot/users", json=payload)
+                else:
+                    self.assertTrue(str(err.status) in self.ctx.send.call_args[0][0])
```

Comment:
It's a good idea to still assert `send()` was awaited before accessing the `call_args`, because if it isn't called `call_args` will be `None`. It's preferred for tests to fail via assertion failures rather than some `TypeError` as the former is easier to understand and fix.